File: estore-ui/src/app/Sizing.ts
export enum Sizing {
	MENS = 'Mens',
	WOMENS = 'Womens',
	KIDS = 'Kids'
}

export function toString(sizing: Sizing): string {
	switch (sizing) {
		case Sizing.MENS:
			return 'Men\'s';
		case Sizing.WOMENS:
			return 'Women\'s';
		case Sizing.KIDS:
			return 'Kids';
		default:
			return '';
	}
}

File: estore-api/src/main/java/com/estore/api/estoreapi/enums/Sizing.java
package com.estore.api.estoreapi.enums;

import com.fasterxml.jackson.annotation.JsonFormat;
import com.fasterxml.jackson.annotation.JsonValue;

@JsonFormat(shape = JsonFormat.Shape.STRING)
public enum Sizing {
    MENS("Mens"),
    WOMENS("Womens"),
    KIDS("Kids");

    private final String unit;

    Sizing(String unit) {
        this.unit = unit;
    }

    @JsonValue
    public String getSizing() {
        return unit;
    }
}
